src/bpftrace.cpp

```diff
@@ -839,7 +839,7 @@ void perf_event_lost(void *cb_cookie, uint64_t lost)
 
 std::vector<std::unique_ptr<AttachedProbe>> BPFtrace::attach_usdt_probe(
     Probe &probe,
-    std::tuple<uint8_t *, uintptr_t> func,
+    BpfProgram &program,
```

Comment:
```suggestion
    BpfProgram &&program,
```
`BpfProgram` (no reference) would work as well, but this expresses the intention better and ensures that the callers std::move into it